app/src/common/shared/org/mozilla/vrbrowser/telemetry/TelemetryWrapper.java

```diff
@@ -184,19 +200,30 @@ public static void addLoadToHistogram(String uri, Long newLoadTime) {
 
         domainMap.add(UrlUtils.stripCommonSubdomains(uriLink.getHost()));
         numUri++;
-        int histogramLoadIndex = toIntExact(newLoadTime / BUCKET_SIZE_MS);
+        int histogramLoadIndex = toIntExact(newLoadTime / LOADING_BUCKET_SIZE_MS);
 
         if (histogramLoadIndex > (HISTOGRAM_SIZE - 2)) {
             histogramLoadIndex = HISTOGRAM_SIZE - 1;
+            Log.e(LOGTAG, "the loading histogram size is overflow.");
```

Comment:
nit: `is overflown`

Reply:
it should be ok to make it to be as a *noun*.